_pages/dev/cors.md

```diff
@@ -0,0 +1,168 @@
+| occ-personalization-id | The `occ-personalization-id` header is required by the personalization feature. If you do not use the personalization feature, this header can be omitted. For more information, see [Configure Personalization for Commerce Web Services](https://help.sap.com/viewer/86dd1373053a4c2da8f9885cc9fbe55d/1905/en-US/e970070f997041c7b3f3e77fcb762744.html) on the SAP Help Portal. |
+| occ-personalization-time | The `occ-personalization-time` header is required by the personalization feature. If you do not use the personalization feature, this header can be omitted. For more information, see [Configure Personalization for Commerce Web Services](https://help.sap.com/viewer/86dd1373053a4c2da8f9885cc9fbe55d/1905/en-US/e970070f997041c7b3f3e77fcb762744.html) on the SAP Help Portal. |
+| x-profile-tag-debug | The `x-profile-tag-debug` header is required by Context-Driven Services. It is used to instruct CDS to produce a trace. If you do not use CDS, this header can be omitted. For more information, see [SAP Commerce Cloud, Context-Driven Services](https://help.sap.com/viewer/product/CONTEXT-DRIVEN_SERVICES/SHIP/en-US?task=discover_task) on the SAP Help Portal. |
+| x-consent-reference | The `x-consent-reference` header is required by Context-Driven Services. If you do not use CDS, this header can be omitted. For more information, see [SAP Commerce Cloud, Context-Driven Services](https://help.sap.com/viewer/product/CONTEXT-DRIVEN_SERVICES/SHIP/en-US?task=discover_task) on the SAP Help Portal. |
+
+### exposedHeaders
+
+Only Context-Driven Services requires a custom header to be exposed, which is the `x-anonymous-consents` header.
+
+### allowCredentials
+
+Request credentials are concerned with cookies, authorization headers, or TLS client certificates. These are not allowed in cross-origin requests by default, which is why a request is blocked when the configuration is not applied.
+
+Spartacus did not send cookies in version 1.x of the Spartacus libraries, but since version 2.0, cookies will be sent for each and every OCC request. This has also been patched to versions 1.4 and 1.5 of the Spartacus libraries.
+
+Sending cookies is required to gain "session infinity", which is also know as "sticky sessions". Session infinity means that the same server behind an API endpoint is used for all subsequent requests for the same session. Although the Commerce API is stateless, there are occasions when multiple parallel calls could fail (for example, add to cart vs get cart). Moreover, the back end will gain performance improvements if requests for the same session are served by the same server.
```

Comment:
We can change the last bit to:

> Although the Commerce API is stateless, there are occasions when multiple parallel or sequential calls could fail. For example, an “add to cart” request followed by a “load cart” request could fail, as the first request could end up at server 1, where as the 2nd request immediately followed after could end up at server 2. The servers might not be fast enough to send around cache invalidation, which is why the 2nd response might fail to catch the added item.

Reply:
This is perfect! Thank you very much!! 🙂